frontend/test/model/common/ValidationErrorIcon.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { ThemeProvider } from '@mui/material/styles'
+import { render, screen, waitFor } from '@testing-library/react'
+import { MarkdownDisplayProps } from 'src/common/MarkdownDisplay'
+import ValidationErrorIcon from 'src/model/beta/common/ValidationErrorIcon'
+import { lightTheme } from 'src/theme'
+import { testAccessRequestSchemaStepNoRender } from 'utils/test/testModels'
+import { describe, expect, it, vi } from 'vitest'
+
+vi.mock('src/common/MarkdownDisplay.tsx', () => ({ default: (_props: MarkdownDisplayProps) => <></> }))
+
+describe('Va;idationError', () => {
+  it('renders a ValidationError component', async () => {
+    render(
+      <ThemeProvider theme={lightTheme}>
+        <ValidationErrorIcon step={testAccessRequestSchemaStepNoRender} />
+      </ThemeProvider>,
+    )
+
+    await waitFor(async () => {
+      expect(await screen.findByLabelText('This step is unfinished')).not.toBeUndefined()
+    })
+  })
+
+  it('does not render a ValidationError component when step is marked as complete', async () => {
```

Comment:
This isn't the name of the component. I also find it's useful to get into the habit of not using the component name to describe the individual test for the following reasons:
- The describe already contains the component name.
- It forces you to be more descriptive with your test naming.

In this case it already is descriptive, but you're repeating the name unnecessarily. Consider renaming to something like `'does not render anything when step is marked as complete'`